flagd-demo/step1.md

```diff
@@ -0,0 +1,19 @@
+## Your Username and Password
+Certain commands will prompt for a username and password.
```

Comment:
Is it possible to set this automatically in the gift config so the user didn't have to worry about it?